torch_ipex/csrc/jit/fusion_pass.cpp

```diff
@@ -323,7 +325,7 @@ class OpFuser {
     }
 
     return std::make_pair(++pos->iterator(), changed);
-}
+  }
```

Comment:
seems like these is a redauant whitespace character here?

Reply:
it is not a redauant whitespace.